lib/stripe/bank_account.ex

```diff
@@ -32,11 +32,18 @@ defmodule Stripe.BankAccount do
     :default_for_currency, :metadata
   ]
 
-  @valid_update_keys [
-    :source, :object, :account_number, :country, :currency,
-    :account_holder_name, :account_holder_type, :routing_number,
-    :default_for_currency, :metadata
-  ]
+  @update_schema %{
+    source: :string,
+    object: :string,
+    account_number: :string,
+    country: :string,
+    currency: :string,
+    account_holder_name: :string,
+    account_holder_type: :string,
+    routing_number: :string,
+    default_for_currency: :string,
+    metadata: :map
+  }
```

Comment:
API docs specify `id`, `default_for_currency` and `metadata` for managed bank account updates.

For other bank account updates, it's `id`, `customer`, `account_holder_name`, `account_holder_type` and `metadata`.

What we have here seems to be a list of all bank account properties. Do we want to handle this later? Seems like you maybe just did a rename for now.